pkg/util/metric/metric.go

```diff
@@ -176,10 +186,261 @@ func maybeTick(m periodic) {
 	}
 }
 
+// A HdrHistogram collects observed values by keeping bucketed counts. For
+// convenience, internally two sets of buckets are kept: A cumulative set (i.e.
+// data is never evicted) and a windowed set (which keeps only recently
+// collected samples).
+//
+// Top-level methods generally apply to the cumulative buckets; the windowed
+// variant is exposed through the Windowed method.
+type HdrHistogram struct {
```

Comment:
Going to assume this is unchanged from before it was removed.

Reply:
Almost! One new function was defined on the `HdrHistogram` to make it adhere to the `IHistogram` interface - `ToPrometheusMetricWindowed()`.